app/views.py

```diff
@@ -184,18 +184,22 @@ def resetpass():
 		return redirect(url_for('resetpass'))
 	return render_template('resetpass.html')
 
-@app.route('/searchevents')
+@app.route('/searchevents', methods=['POST'])
 def searchevents():
 	"""A route to search events depending on the events category or location"""
-	parameter = request.args.get('parameter', None)
-	if parameter == None:
-		return jsonify(status="no event")
+	parameter = request.form['search']
+	if len(parameter.strip()) == 0:
```

Comment:
* bad practice to use Len() as condition value, use use pylint to follow best practices.